.vscode/settings.json

```
@@ -0,0 +1,5 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
```

vscode says it should be string for me.

```suggestion
    "source.fixAll.eslint": "always"
```

---

Works!